src/main/kotlin/no/skatteetaten/aurora/mokey/model/ApplicationData.kt

```diff
@@ -94,10 +96,22 @@ data class ManagementEndpointResult<T>(
         get() = resultCode == "OK"
 }
 
+private val logger = KotlinLogging.logger {}
+const val jsonFormatErrorMsg = """{ "error": "Received content is not json" }"""
+
 data class HttpResponse(
     val content: String,
     val code: Int
-)
+) {
+    fun jsonContentOrErrorMsg() =
+        try {
+            jacksonObjectMapper().readTree(content)
+            content
+        } catch (ignored: Exception) {
+            logger.warn { "Response is not json format: $content" }
+            jsonFormatErrorMsg
```

Comment:
Bør vi sende med litt av content eller hele?

Reply:
Det er et godt spm, for dette kan vel potensielt bli ganske mye??

Reply:
Hva om vi gjør noe slikt:
```
${content.take(100)}
```